web/main.py

```diff
@@ -45,7 +45,14 @@ def handler(dataset=None, filename=None):
     if blob is None:
         abort(404)
 
-    return blob.download_as_text()
+    response = Response(blob.download_as_bytes())
```

Comment:
Not that this changes the previous implementation, but this will always load files into memory before serving them. Probably not bad most of the time, but I see this details being forgotten, and trying to download a larger file causing problems.

Reply:
We could chunk the response here, similar to what we're doing in seqr. But I hope we're not serving hundreds of MBs this way ;)

Reply:
I saw [this on StackOverflow](https://stackoverflow.com/questions/50380237/reading-really-big-blobs-without-downloading-them-in-google-cloud-streaming), about downloading to a stream, and using that as the response:

```python
def getStream(blob):
    stream = open('myStream','wb', os.O_NONBLOCK)
    streaming = blob.download_to_file(stream)
```